### PR TITLE
[Ready]Adds the Pax reagent, small tweaks to pacifism

### PR DESCRIPTION
**Pull request by XDTM**
:cl: XDTM
add: Added a new reagent: Pax.
add: Pax is made with Mindbreaker, Synaptizine and Water, and those affected by it are forced to be nonviolent against living beings. At least, not directly.
/:cl:

Additionally added a feedback message when trying to hit stuff under pacifism; i didn't include it last time since it was spammy, but now that we have the (x number) system there's no reason not to have it.

### CODE REVIEW

**Comment by Potato-Masher** [comment]
Cook this up with some mute toxin or something and the only thing they can do is run. I like it.

**Comment by Armhulen** [comment]
Make it send good vibe messages :)

**Comment by Robustin** [comment]
I'd make it disable any intent besides help since you can kill people fairly easily with grab intent and disarm also lets you swat away someone's weapon, grab it, and then attack them with it.

It might actually be useful with forced help intent, blocking harm intent would barely do anything.

**Comment by Armhulen** [comment]
harm and grab, disarm can stay

**Comment by XDTM** [comment]
grab stays, but you cannot grab people aggressively, only pull

**Comment by Robustin** [comment]
If you keep disarm, this chem joins the long list of useless meme chems. Disarm makes this chem much more interesting and actually potentially useful.

**Comment by PKPenguin321** [comment]
misnomer

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -684,7 +684,7 @@
 
 	if(ishuman(user) && prob(40) && src.density)
 		var/mob/living/carbon/human/H = user
-		if((H.disabilities & DUMB) && Adjacent(user))
+		if((H.has_disability(DEAF)) && Adjacent(user))
```
Comment (SpaceManiac) [inline code/game/machinery/doors/airlock.dm]: This should still be `DUMB`, I think?

Comment (XDTM) [reply code/game/machinery/doors/airlock.dm]: Dammit

**Comment by Tlaltecuhtli** [comment]
can you give pk borgs the reagent (maybe lock behind """"""""upgrades""""""")

**Comment by kevinz000** [comment]
@Tlaltecuhtli out of the question.
also why is pax reagent something that makes you peaceful this is literally the exact opposite mate. reee.

**Comment by kevinz000** [comment]
@XDTM can you make the fix pr separate?
because everyone slamming into doors is not exactly a low priority issue.
id' do it myself because it's unplayable with the headbutts but i'm lazy and i'm not playing soo :shrug:

**Comment by praisenarsie** [comment]
insert bryce pax joke here

**Comment by FantasticFwoosh** [comment]
So basically you're going to mix this to create killer Pax reagent bees you cannot retaliate to.

**_OK_** 🌝

File: code/datums/brain_damage/mild.dm [hunk 2]
```diff
@@ -54,7 +54,7 @@
 	..()
 
 /datum/brain_trauma/mild/dumbness/on_lose()
-	owner.remove_disability(DUMB, TRAUMA_DISABILITY)
+	owner.remove_disability(DUMB, list(TRAUMA_DISABILITY))
```
Comment (ShizCalev) [inline code/datums/brain_damage/mild.dm]: undo these for remove_disability to keep them consistent with add_disability.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/datums/brain_damage/mild.dm (hunk 2 above)
Comment (XDTM) [reply code/datums/brain_damage/mild.dm]: Alright

**Comment by XDTM** [comment]
@ShizCalev done

**Comment by XDTM** [comment]
I don't think the travis error is on my part here

**Comment by octareenroon91** [comment]
Forcing help intent is still relevant to blobs right?

**Comment by XDTM** [comment]
Forcing help intent happens regularly, but i think blobs aren't protected as they're structures and not mobs, and i'd rather not make a special exception to attack_obj for them.

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 3]
```diff
@@ -1688,3 +1688,23 @@
...
+	description = "A colorless liquid that suppresses violence on the subjects."
+	color = "#AAAAAA55"
+	taste_description = "water"
+	metabolization_rate = 0.25 * REAGENTS_METABOLISM
+
+/datum/reagent/pax/on_mob_add(mob/M)
+	..()
+	if(isliving(M))
+		var/mob/living/L = M
+		L.add_disability(PACIFISM, CHEMICAL_DISABILITY)
```
Comment (ShizCalev) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: DISABILITY_PACIFISM

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 4]
```diff
@@ -1688,3 +1688,23 @@
...
+
+/datum/reagent/pax/on_mob_delete(mob/M)
+	if(isliving(M))
+		var/mob/living/L = M
+		L.remove_disability(PACIFISM, CHEMICAL_DISABILITY)
```
Comment (ShizCalev) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: DISABILITY_PACIFISM

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Pull request merged**